lib/project_types/extension/messages/messages.rb

```diff
@@ -89,6 +89,10 @@ module Messages
         help: <<~HELP,
           Push the current extension to Shopify.
             Usage: {{command:%s extension push}}
+            Options:
+              {{command:--api-key=API_KEY}} The API key used to register an app with the extension. This can be found on the app page on Partners Dashboard.
```

Comment:
cc @MeredithCastile

Reply:
Just to be clear — I understand this as a short term solution. I really don't want to have "extension" be a term for both "scripts" and "ui" in the long term. cc: @claudiomunoz 

___

Digging into this pre-Unite solution, then: 

- I've noticed in research calls that partner devs don't always know the name of the Partners Dashboard.

- Most of our "options" help text is, I believe, verb based (ex: "Output verbose messages on internal operations").

- I have a larger question around "options" that aren't all that optional. (This command always requires an API key, no?) I'm wondering if "flags" is a clearer noun to use across the board than "options"? Worth discussing in January.

___

For the sake of giving you _something_, though, @pepicrft and @amcaplan, here's an attempt at phasing that uses an imperative verb and avoids "Partner Dashboard":
`Connect your extension and app by inserting your app's API key (which you can get from your app setup page on shopify.dev).`

Just to be clear: the app API key is the one they'd use — not the API secret key?

![image](https://user-images.githubusercontent.com/51347120/146607883-b99fb42d-a5ec-4bb5-93d6-a33bbffe5fde.png)